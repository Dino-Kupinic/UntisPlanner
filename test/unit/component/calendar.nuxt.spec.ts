import {describe, expect, it} from "vitest"
import {Calendar} from "#components"
import {mountSuspended} from "@nuxt/test-utils/runtime"
import {before} from "node:test"

let component = "";

describe('Calendar', () => {
  before(async () => {
    component = await mountSuspended(Calendar);
  });

  it('is mounted properly', () => {
    expect(component).toMatchSnapshot();
  });
});

