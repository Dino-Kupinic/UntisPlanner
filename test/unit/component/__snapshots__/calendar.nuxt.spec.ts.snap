// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Calendar > is mounted properly 1`] = `""`;
